pkg/application/doguInstallationUseCase.go

```diff
@@ -10,21 +11,22 @@ type DoguInstallationUseCase struct {
 	doguDomainUseCase domainservice.DoguInstallationDomainUseCase
 }
 
-func (useCase *DoguInstallationUseCase) validateDoguHealth() error {
-	installedDogus, err := useCase.doguRepo.GetAll()
+func (useCase *DoguInstallationUseCase) validateDoguHealth(ctx context.Context) error {
+	//TODO: this is only a stub to get an idea of the upcoming implementation
+	installedDogus, err := useCase.doguRepo.GetAll(ctx)
 	if err != nil {
 		return fmt.Errorf("cannot evaluate dogu health states: %w", err)
 	}
 
 	return useCase.doguDomainUseCase.ValidateDoguHealth(installedDogus)
 }
 
-func (useCase *DoguInstallationUseCase) installDogu(dogu string, version string) error {
+func (useCase *DoguInstallationUseCase) installDogu(ctx context.Context, doguName string, version string) error {
```

Comment:
🚫 **[checkstyle]** <unused> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>func `(*DoguInstallationUseCase).installDogu` is unused

Reply:
This is a stub and is needed later on.